core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -188,7 +188,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
       }
     }
 
-    def abortActiveWorkflows(submissionId: UUID) = {
+    def abortActiveWorkflows(submissionId: UUID): Future[Seq[Future[(Option[String], Try[ExecutionServiceStatus])]]] = {
```

Comment:
Well I'd still advocate fixing this method to have a sane type signature, and if we don't want to wait for the results before checking for status then just don't call it in a for comprehension below.

My attempt at fixing the method:
```
    def abortActiveWorkflows(submissionId: UUID): Future[Seq[Try[ExecutionServiceStatus]]] = {
      datasource.inTransaction { dataAccess =>
        // look up abortable WorkflowRecs for this submission
        dataAccess.workflowQuery.findWorkflowsForAbort(submissionId).result
      }.flatMap { workflows =>
        Future.sequence(workflows.map { workflow =>
          executionServiceCluster.abort(workflow, UserInfo.buildFromTokens(credential))
        })
      }
    }
```